help/destinations/catalog/crm/salesforce.md

```diff
@@ -0,0 +1,161 @@
+---
+keywords: crm;CRM;crm destinations;salesforce crm;salesforce crm destination
+title: Salesforce CRM connection
+description: The Salesforce CRM destination allows you to export your account data and activate it within Salesforce CRM for your business needs.
+---
+
+# [!DNL Salesforce CRM] connection
+
+## Overview {#overview}
+
+[Salesforce CRM](https://www.salesforce.com/) is a popular Customer Relationship Management (CRM) platform.
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [Salesforce REST API](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/dome_composite_upsert_example.htm?q=contacts), which allows you to update identities within a segment into Salesforce CRM.
+
+## Destination specifics {#specifics}
+
+Note the following details that are specific to the [!DNL Salesforce CRM] destination:
+
+* [!DNL Adobe Experience Platform] segments are exported to [!DNL Salesforce CRM] using a custom `AdobeExperiencePlatformSegments` field. Instructions are further below, in the [Prerequisites](#prerequisites) section.
```

Comment:
Let's be a bit more specific here. Instead of "Instructions are further below, in the [Prerequisites](#prerequisites) section.", let's call out instructions to do what exactly.

Reply:
@vgiurgiu Wouldn't that create content duplication as I'd be adding the below

```
* Create `AdobeExperiencePlatformSegments` as a custom field to allow Experience Platform to update your profiles within Salesforce CRM. Refer to the Salesforce documentation to [create custom fields](https://help.salesforce.com/s/articleView?id=sf.adding_fields.htm&type=5) if you need additional guidance.

    | Field | Data Type |
    |---|---|
    | `AdobeExperiencePlatformSegments` | `Text Area Long` |
```
And then within prerequisites I would have to add a preamble *If you have not already done so,*

```
1. If you have not already done so, create `AdobeExperiencePlatformSegments` as a custom field to allow Experience Platform to update your profiles within Salesforce CRM. Refer to the Salesforce documentation to [create custom fields](https://help.salesforce.com/s/articleView?id=sf.adding_fields.htm&type=5) if you need additional guidance.

    | Field | Data Type |
    |---|---|
    | `AdobeExperiencePlatformSegments` | `Text Area Long` |

```

Would rewording it as below work ?

* [!DNL Adobe Experience Platform] segments are exported to [!DNL Salesforce CRM] using a custom field. The details for which are further below, in the [Prerequisites](#prerequisites) section.

Reply:
Sure, will update to the same.